src/ClusterBootstrap/deploy.py

```diff
@@ -3577,6 +3628,17 @@ def run_command( args, command, nargs, parser ):
             print "Error: kubernetes need a subcommand."
             exit()
 
+    elif command == "gpulabel":
+        kubernetes_label_GpuTypes()
+
+    elif command == "gen_scripts":
```

Comment:
current runscript function should render the scripts file with jinja2, you can skip the generation process.